Author: <csmall@debian.org>
Description: Renames second SZ to SIZE,  means we don't follow SCO but less confusion
Bug-Debian: http://bugs.debian.org/541061
Index: b/ps/output.c
===================================================================
--- a/ps/output.c	2009-11-24 21:00:47.000000000 +1100
+++ b/ps/output.c	2009-11-24 21:00:48.000000000 +1100
@@ -1522,7 +1522,7 @@
 {"sigcatch",  "CAUGHT",  pr_sigcatch, sr_nop,     9,   0,    XXX, TO|SIGNAL}, /*caught*/
 {"sigignore", "IGNORED", pr_sigignore,sr_nop,     9,   0,    XXX, TO|SIGNAL}, /*ignored*/
 {"sigmask",   "BLOCKED", pr_sigmask,  sr_nop,     9,   0,    XXX, TO|SIGNAL}, /*blocked*/
-{"size",      "SZ",      pr_swapable, sr_swapable, 5,  0,    SCO, PO|RIGHT},
+{"size",      "SIZE",      pr_swapable, sr_swapable, 5,  0,    SCO, PO|RIGHT},
 {"sl",        "SL",      pr_nop,      sr_nop,     3,   0,    XXX, AN|RIGHT},
 {"spid",      "SPID",    pr_thread,   sr_tid,     5,   0,    SGI, TO|PIDMAX|RIGHT},
 {"stackp",    "STACKP",  pr_stackp,   sr_start_stack, 8, 0,  LNX, PO|RIGHT}, /*start_stack*/
Index: b/ps/ps.1
===================================================================
--- a/ps/ps.1	2009-11-24 21:00:47.000000000 +1100
+++ b/ps/ps.1	2009-11-24 21:00:48.000000000 +1100
@@ -1254,7 +1254,7 @@
 see\ \fBblocked\fR.  (alias\ \fBblocked\fR,\ \fBsig_block\fR).
 T}
 
-size	SZ	T{
+size	SIZE	T{
 approximate amount of swap space that would be required
 if the process were to dirty all writable pages and then
 be swapped out.
